### PR TITLE
js-hotfix-nested-list

### PR DESCRIPTION
**Pull request by sampoonachot**
there was some styling in `root.less` for `ul` and `li` that was applying to `nested-list.less` so I removed that.

### CODE REVIEW

File: client/styling/root.less [hunk 1]
```diff
@@ -64,13 +64,6 @@ textarea {
   resize: none;
 }
 
-ul, li {
```
Comment (rawfalafel) [inline client/styling/root.less]: 👍 Do you know where the other rules above come from? We probably want to remove those too, if they're interfering with other BEM styles

File: client/styling/nested-list.less [hunk 2]
```diff
@@ -10,83 +10,26 @@
...
     .nested-list-link();
     padding: 8px;
 
-    &::before {
-      content: "\E308";
-      font-family: "MMS Icons";
-      color: @gray5;
-      margin-right: 5px;
-    }
+    // &::before {
```
Comment (rawfalafel) [inline client/styling/nested-list.less]: Did you mean to delete this?

File: client/components/ui-design-system/index.jsx [hunk 3]
```diff
@@ -10,36 +10,35 @@ const UIDesignSystem = ({ children }) => (
   <nav className="navigation-sidebar">
     <ul className="navigation navigation-is-vertical navigation-is-vertical-parent">
       <li><h4>Product Design</h4>
-       <ul className="navigation-is-vertical-parent">
-        <li><Link to='/ui-design-system/welcome' className="navigation-link">Welcome</Link></li>
-       </ul>
+        <ul className="navigation-is-vertical-parent">
```
Comment (rawfalafel) [inline client/components/ui-design-system/index.jsx]: the indentation looks weird 👀

File: client/styling/nested-list.less (hunk 2 above)
Comment (sampoonachot) [reply client/styling/nested-list.less]: yeah wanted to keep it in until i merged the mms-icons

File: client/components/ui-design-system/index.jsx (hunk 3 above)
Comment (sampoonachot) [reply client/components/ui-design-system/index.jsx]: i fixed it in the recent commit

File: client/styling/root.less (hunk 1 above)
Comment (sampoonachot) [reply client/styling/root.less]: `pre` is being used for the code snippets.. `img` in multiple files?

Comment (rawfalafel) [reply client/styling/root.less]: I think the styles in `input` might be overwriting the styles [here](https://github.com/leafygreen/design/blob/master/client/styling/forms.less#L2)

Comment (sampoonachot) [reply client/styling/root.less]: 👍

**Pull request merged**